packages/playground/src/shared/image/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * A super basic Turtle geometry renderer hooked up to Theatre, so the parameters
+ * can be tweaked and animated.
+ */
+import {getProject, types} from '@theatre/core'
+import studio from '@theatre/studio'
+import React, {useEffect, useState} from 'react'
+import {render} from 'react-dom'
+import styled from 'styled-components'
+
+studio.initialize()
+const project = getProject('Image type playground', {
+  assets: {
+    baseUrl: 'http://localhost:3000',
```

Comment:
detect `baseUrl`

Reply:
What do you mean by this?

Reply:
The `baseUrl` wouldn't work if the host isn't `localhost:3000`

Reply:
Ah, you meant it specifically for the playground. Not sure how we could detect it, or if it was even worth it, since it's just a playground, people can modify it as they please.